data/card_catalogue/triggers.json

```diff
@@ -5,11 +5,11 @@
     "tier": "green",
     "details": {
       "rules": {
-        "passive": "Attacks against characters who love you deal 25 bonus damage.",
-        "active": "A visible character develops feelings for you."
+        "active": "A visible NPC develops feelings for you.",
+        "passive": "Attacks against characters who love you deal 25 bonus damage."
```

Comment:
A prime candidate for a non-player only clause

Reply:
Let's do it for consistently, but this effect would never work on a player since they get to decide if the condition is true